boards/weact_f401ccu6/src/main.rs

```diff
@@ -0,0 +1,430 @@
+//! Board file for WeAct STM32F401CCU6 Core Board
+//!
+//! - <https://github.com/WeActTC/MiniF4-STM32F4x1>
+
+#![no_std]
+// Disable this attribute when documenting, as a workaround for
+// https://github.com/rust-lang/rust/issues/62184.
+#![cfg_attr(not(doc), no_main)]
+#![deny(missing_docs)]
+
+use capsules::virtual_alarm::VirtualMuxAlarm;
+use components::gpio::GpioComponent;
+use kernel::capabilities;
+use kernel::common::dynamic_deferred_call::{DynamicDeferredCall, DynamicDeferredCallClientState};
+use kernel::component::Component;
+use kernel::hil::led::LedLow;
+use kernel::Platform;
+use kernel::{create_capability, debug, static_init};
+
+use stm32f401cc::interrupt_service::Stm32f401ccDefaultPeripherals;
+
+/// Support routines for debugging I/O.
+pub mod io;
+
+// Unit tests
+#[allow(dead_code)]
+mod multi_alarm_test;
+
+// Number of concurrent processes this platform supports.
+const NUM_PROCS: usize = 4;
+
+// Actual memory for holding the active process structures.
+static mut PROCESSES: [Option<&'static dyn kernel::procs::ProcessType>; NUM_PROCS] =
+    [None, None, None, None];
+
+static mut CHIP: Option<&'static stm32f401cc::chip::Stm32f4xx<Stm32f401ccDefaultPeripherals>> =
+    None;
+
+// How should the kernel respond when a process faults.
+const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultResponse::Panic;
+
+/// Dummy buffer that causes the linker to reserve enough space for the stack.
+#[no_mangle]
+#[link_section = ".stack_buffer"]
+pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
+
+/// A structure representing this platform that holds references to all
+/// capsules for this platform.
+struct WeactF401CC {
+    console: &'static capsules::console::Console<'static>,
+    ipc: kernel::ipc::IPC,
```

Comment:
```suggestion
    ipc: kernel::ipc::IPC<NUM_PROCS>,
```
I think this change will fix the CI. Changes were recently merged into master that makes this necessary. You may want to rebase the PR on master first.